dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -419,6 +443,29 @@ export const Card = ({
 			</Link>
 		);
 
+	const MediaPill = () => (
```

Comment:
Could the pill work be lifted out of this PR into is own branch?

Reply:
Sorry, that was a rebase gone awry! The pill changes are in a separate feature branch which this is being merged into. I've rebased that from `main` and then this from that branch so it's only showing the waveform changes now.